features/general.feature

```diff
@@ -71,7 +71,7 @@ Feature: General tests of WP Launch Check
 
   Scenario: WordPress has a new major version but no new minor version
     Given a WP install
-    And I run `wp core download --version=5.6.11 --force`
+    And I run `wp core download --version=4.6.26 --force`
```

Comment:
The 6.3 update looks fine but this is curious. Why would we drop to 4.x and not just update to the lastest 5.x patch?